basic_network.go

```diff
@@ -103,6 +103,10 @@ func (n *BasicVideoNetwork) GetBroadcaster(strmID string) (lpnet.Broadcaster, er
 	return b, nil
```

Comment:
We remove any place where we are writing a message into an in-coming stream.  Instead, we get a (outgoing)stream from the node and send the message using that.